src/stories/Accordion.stories.mdx

```diff
@@ -0,0 +1,47 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import Accordion from './Accordion';
+
+<Meta title="Example/Accordion" component={Accordion} />
```

Comment:
Eventually I'd like to remove all of the examples. Let's put this in a new `Components` directory for now.
```suggestion
<Meta title="Components/Accordion" component={Accordion} />
```